devel/rizin/Portfile

```diff
@@ -34,12 +36,12 @@ depends_lib         port:capstone \
                     port:libuv
 
 configure.args-append \
-    -Duse_sys_capstone=enabled \
-    -Duse_sys_magic=disabled \
-    -Duse_sys_libzip=enabled \
-    -Duse_sys_zlib=enabled \
-    -Duse_sys_lz4=enabled \
-    -Duse_sys_xxhash=disabled \
-    -Duse_sys_openssl=enabled \
-    -Duse_sys_tree_sitter=enabled \
-    -Duse_sys_libuv=enabled
+                    -Duse_sys_capstone=enabled \
+                    -Duse_sys_magic=disabled \
+                    -Duse_sys_libzip=enabled \
+                    -Duse_sys_zlib=enabled \
+                    -Duse_sys_lz4=enabled \
+                    -Duse_sys_xxhash=disabled \
+                    -Duse_sys_openssl=enabled \
+                    -Duse_sys_tree_sitter=enabled \
+                    -Duse_sys_libuv=enabled
```

Comment:
Can you please avoid making version updates and trivial whitespace changes in the same commit. By all means do both in the same MR but use seperate commits for each.

Reply:
As requested, version update and whitespace changes have been split into separate commits.